commercial/app/controllers/PrebidAnalyticsController.scala

```diff
@@ -0,0 +1,30 @@
+package commercial.controllers
+
+import common.Logging
+import conf.switches.Switches.prebidAnalytics
+import model.Cached.WithoutRevalidationResult
+import model.{CacheTime, Cached, TinyResponse}
+import play.api.libs.json.JsValue
+import play.api.libs.json.Json.prettyPrint
+import play.api.mvc._
+
+class PrebidAnalyticsController(val controllerComponents: ControllerComponents) extends BaseController with Logging {
+
+  private def serve404[A](implicit request: Request[A]) =
+    Cached(CacheTime.NotFound)(WithoutRevalidationResult(NotFound))
+
+  def insert(): Action[JsValue] = Action(parse.json) { implicit request =>
+    if (prebidAnalytics.isSwitchedOn) {
+      log.info(prettyPrint(request.body))
```

Comment:
So are we going to collect our usage stats through the ELK stack?

Reply:
Initially.  This is stage 1 in a 7 stage process.  To prove the analytics gathering process with a small sample.